src/app/layout/account-switcher.component.ts

```diff
@@ -105,6 +107,11 @@ export class AccountSwitcherComponent implements OnInit {
       if (userId == null) {
         continue;
       }
+
+      if (userId === (await this.stateService.getUserId())) {
+        continue;
+      }
+
```

Comment:
I recommend merging this conditional with the one above it: `(userId == null || userId == activeUserId)`